benchmarks/wiki/README.md

```diff
@@ -0,0 +1,15 @@
+## Cohere wiki text embedding benchmark
+a dataset with 35M 768D vectors
+
+### Generating data
+run
+```bash
+python datagen.py
+```
+to generate the lance dataset
+
+run
+```bash
+python gen_ground_truth.py
```

Comment:
There is no `gen_ground_truth` file, btw, `datasets.load_dataset()` will just download the dataset once and cached on your machine?
So `datagen.py` will do the downloading anyway?

Reply:
i need to rename it from `test.py`. still debugging that script